pyspline/pyCurve.py

```diff
@@ -894,7 +894,7 @@ def writeIGES_parameters(self, handle, Pcount, counter):
                 counter += 1
                 pos_counter = 0
 
-        for i in range(self.nCtl):
+        for _i in range(self.nCtl):
```

Comment:
This is minor, but what is the motivation for this indexing change?

Reply:
Flake8 checks for unused loop variables, and says they should be preceded by an underscore to show that it's intentionally unused.